.github/workflows/schema.yml

```diff
@@ -28,4 +28,4 @@ jobs:
 
```

Comment:
![image](https://github.com/prefix-dev/pixi/assets/29506042/80da711b-382b-4559-aa33-460ea76a201a)
this should be a 

```yml
- uses: prefix-dev/setup-pixi
  with:
    pixi-version: v0.17.1
    environments: schema
```

Reply:
I actually think we should dog-food ourselfs more so we should always use latest now you point this out.